coalib/output/Logging.py

```diff
@@ -60,15 +62,19 @@ def configure_logging():
         'formatters': {
             'colored': {
                 '()': 'colorlog.ColoredFormatter',
-                'format': '%(log_color)s[%(levelname)s]%(reset)s[%(asctime)s]'
-                          ' %(message)s',
+                'format': '%(log_color)s[%(levelname)s]%(reset)s'
+                          '[%(asctime)s] %(message)s',
```

Comment:
hmm if possible remove the change or make a different commit, it's a layout change ;) Also the previous layout is valid, so it's more personal preference ;)